File: snippet/query-examples/clojure/har-get.clj
(require '[clj-http.client :as client])

(client/get "https://will-x86.com/api/pixelsq" {:headers {:User-Agent "curl/7.68.0"
                                                          :Accept "*/*"}
                                                :query-params {:x "1"
                                                               :y "2"
                                                               :hex "#102932"}})

File: snippet/query-examples/python/har-get.py
import http.client

conn = http.client.HTTPSConnection("will-x86.com")

headers = {
    'User-Agent': "curl/7.68.0",
    'Accept': "*/*"
    }

conn.request("GET", "/api/pixelsq?x=1&y=2&hex=%23102932", headers=headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))
